cishouseholds/pipeline/pipeline_stages.py

```diff
@@ -374,7 +374,7 @@ def process_soc_data(
     Process soc data and combine result with survey responses data
     """
     join_on_columns = ["work_main_job_title", "work_main_job_role"]
-    window = Window.partitionBy(*join_on_columns)
+    window = Window.partitionBy("standard_occupational_classification_code", *join_on_columns)
     inconsistences_resolution_df = extract_from_table(inconsistences_resolution_table).withColumnRenamed(
```

Comment:
Could we edit the file that's used to create this table, so that the column name is already `resolved_soc_code`, so it doesn't need editing each run?